node_modules/oae-principals/lib/api.user.js

```diff
@@ -181,11 +183,12 @@ var createUser = module.exports.createUser = function(ctx, tenantAlias, displayN
  *
  * @param  {Context}    ctx                         Standard context object containing the current user and the current tenant
  * @param  {User}       user                        The user to create
+ * @param  {String}     [email]                     The email address of the user
```

Comment:
I agree with this approach, but it's not immediately obvious. It might be worth documenting here why `email` is separate from the user profile that is being persisted.